js-joda/build.boot

```diff
@@ -0,0 +1,29 @@
+(set-env!
+  :resource-paths #{"resources"}
+  :dependencies '[[cljsjs/boot-cljsjs "0.5.2"  :scope "test"]])
+
+(require '[cljsjs.boot-cljsjs.packaging :refer :all])
+
+(def +lib-version+ "1.3.0")
+(def +version+ (str +lib-version+ "-0"))
+
+(task-options!
+ pom  {:project     'cljsjs/js-joda
+       :version     +version+
+       :description "A date and time library for javascript"
+       :url         "https://js-joda.github.io/js-joda/"
+       :scm         {:url "https://github.com/cljsjs/packages"}
+       :license     {"MIT" "http://opensource.org/licenses/MIT"}})
+
+(deftask package []
+  (comp
+    #_(download :url (format "https://github.com/js-joda/js-joda/archive/v%s.zip" +lib-version+)
```

Comment:
Please download the file here instead of including it in the repo.

If the file needs some small changes to work, `replace-content` task might help: https://github.com/cljsjs/packages/blob/42a0f19c36ab89a36174ee620fb61900ee89c001/react-input-mask/build.boot#L29